### ISSUE TITLE
Input clarifications

### ISSUE THREAD

**Issue by jrhbcn** [Berkays/ReflowPlate#1]
Hi, 

First, this project looks incredible cool, thanks a ton for sharing it! I am planning to build it myself, however, I have some questions I need some help before trying to build it. 

1. I have a 15V-6A power supply I would like to use (instead of a 12V) one. I see that both the AMS1117 and the WSP4409A are within specifications. Is it safe to use 15V as input? or is there some other implications I don't see in the schematics?
2. I see that the NTC uses a 10k resistor for the voltage divider. However, In the firmware `THERMISTOR_SERIES_RESISTOR` is set to 4,700. Shouldn't it be set to 10,000? In any case, I am planning to use a 100k NTC 3950 thermistor so I guess I need to also change `THERMISTOR_NOMINAL_RESISTOR` to 100,000 and `TEMPERATURE_METHOD` to `NTC_STEINHART` and recompile right?
3. I have never used aluminium plates and I am a little nervous dealing with them. I would like to avoid soldering into them in case I hit the aluminium or it desolders when heating the plate. Therefore, I have increased the pad size and specially the solder mask size for the pads. You can see my noob attempt attached here [1-PCB_Plate.json](https://github.com/Berkays/ReflowPlate/files/13259534/1-PCB_Plate.json) (json file that can be opened with easyeda). Do you think this is better or in your experience or I am just being paranoid? My plan is to use ring like crimp connectors with some m2 screws or, if that is not possible, solder but with more space with the extra solder mask. 

I am sure I'll find more questions later, thanks anyway! ;)

**Comment by Berkays** [comment]
Hello,

> I have a 15V-6A power supply I would like to use (instead of a 12V) one. I see that both the AMS1117 and the WSP4409A are within specifications. Is it safe to use 15V as input? or is there some other implications I don't see in the schematics?

It should be safe to use with 15V. If you are using a higher power display, monitor the regulator temperature. Also FAN output uses VCC so 12V might not be ok if you are using that feature.

> I see that the NTC uses a 10k resistor for the voltage divider. However, In the firmware THERMISTOR_SERIES_RESISTOR is set to 4,700. Shouldn't it be set to 10,000? In any case, I am planning to use a 100k NTC 3950 thermistor so I guess I need to also change THERMISTOR_NOMINAL_RESISTOR to 100,000 and TEMPERATURE_METHOD to NTC_STEINHART and recompile right?

You are right. Series resistor should be 10k and for 100k NTC set `THERMISTOR_NOMINAL_RESISTOR` to 100k. Also correct about `TEMPERATURE_METHOD`

About the Aluminium plate, I don't have much experience either apart from this project. It is probably a good idea to increase pad size. More you have isolation between the connector and the heat source the better but there isn't much we can do as the plate conducts heat pretty effectively.

BTW, I made some minor adjustments under the version 1.1

**Comment by jrhbcn** [comment]
Thank you very much for your answers @Berkays,

> It should be safe to use with 15V. If you are using a higher power display, monitor the regulator temperature. Also FAN output uses VCC so 12V might not be ok if you are using that feature.

Great, I saw than the firmware code has some variables set to 12V (like [here](https://github.com/Berkays/ReflowPlate/blob/41cd59a7f580ccb82575c403b6d78afc40fbd24a/Firmware/src/screens/AutomaticModeScreen.h#L23) for instance). Do I also need to change them to 15V or it is just initialization and actual values are read from ADC? 
 
> BTW, I made some minor adjustments under the version 1.1

Perfect! I also wanted to add a status LED and you just added it. ;) Do you plan to release a new firmware with it? Something like blinking when reflow is ON and maybe stay on when finished and plate still hot (>40º)?

**Comment by Berkays** [comment]
Yes I'll update the source when i have the time. 12V set variables is actually important for current limiting. I can also change them to a precompiled value. You can replace them with 15 in the meantime.

**Comment by aliyoruk38** [comment]
Hi
When I use 100k ntc, there is a lot of deviation in temperature measurement when I exceed 160 degrees.
I use TEMPERATURE_METHOD as NTC_STEINHART
Is it possible to make calculations using the °C Ohms table for 100k ntc?
Can you help me with the changes I need to make in the code regarding this issue?
